### DISCUSSION TITLE
Update The Dark or Light Mode Colors

### DISCUSSION THREAD

**Discussion by coderchef26** [mui/toolpad#4470] [Help and support] [answered]
I have just started to use toolpad, and I was wondering using MUI's theming capabilities how can I change the colors of the background and defaults for dark or light mode.

**Comment by bharatkashyap** [comment, marked as answer]
Here's an example showcasing how to deeply customise the default theme: https://mui.com/toolpad/core/react-sign-in-page/#theme You can use all customisation options mentioned in the Material UI theme customisation docs: https://mui.com/material-ui/customization/theming/
